NEWS

```diff
@@ -10,6 +10,12 @@ CHANGES WITH 256 in spe:
           section, then all assigned VLAN IDs on the interface that are not
           configured in the .network file are removed.
 
+        * systemd-gpt-auto-generator will stop generating units for ESP or
+          XBOOTLDR partitions if it founds mount entries in the /boot/ or /efi/
+          hierarchies in fstab. This is to prevent the generator from
+          interfering with systems where ESP is mounted on /boot/efi/. This
```

Comment:
where the ESP is explicitly configured to be mounted at some path, for example /boot/efi/